mariadb_copy.md

```diff
@@ -10,6 +10,8 @@ cluster.
 
   * The OpenStackControlPlane resource must be already created at this point.
 
+  * The MariaDB PVC is created
```

Comment:
This is a bit superfluous as below we have "MariaDB is running" prerequisite. If it can't create PVC, it won't be running (the issue you've hit).

Rather than focusing on PVC of MariaDB, let's focus on having free PVs. So i think we don't have to mention anything additional in "MariaDB copy" or "OpenStack Control Plane Deployment" documents, but we should add a prerequisite into the "Backend services deployment" that we need free PVs so that MariaDB and Rabbit can create their PVCs. AFAIK the `make crc_storage` step creates PVs, not PVCs.

Reply:
yeah, I reprovisioned CRC, and I think I understand now:
the PVC is already there, but it is pending
```
[dpadev@osp-dev-05 install_yamls]$ oc get pvc mariadb-openstack -o jsonpath='{.status.phase}{"\n"}'                                                                                           
Pending         
```
when we run `make crc_storage`  it creates the PVs, and then the `openstack-db-init ` pod bounds the PVC:
```
[dpadev@osp-dev-05 install_yamls]$ oc get pvc mariadb-openstack -o jsonpath='{.status.phase}{"\n"}'                                                                                           
Bound         
```